pkg/test/framework/components/policybackend/kube.go

```diff
@@ -257,13 +256,6 @@ func (c *kubeComponent) ID() resource.ID {
 	return c.id
 }
 
-func (c *kubeComponent) Reset() error {
```

Comment:
This may cause some Mixer tests to misbehave, as there might be a single instance that is shared among multiple tests.

Reply:
AFAICT nothing was calling this. Did I miss a call site?